driver-core/pom.xml

```diff
@@ -201,7 +201,7 @@
                         <_include>-osgi.bnd</_include>
                         <Import-Package>
                             <!-- JNR does not provide OSGi bundles, so exclude it; the driver can live without it -->
-                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,*]]></Import-Package>
+                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,!sun.misc.*,*]]></Import-Package>
```

Comment:
I'm unsure if this is the right change, but this was needed to get the OSGi tests to work appropriately.